Logger/LogFactory.cs

```diff
@@ -1,11 +1,33 @@
 ﻿namespace Logger
 {
-    public class LogFactory
+    using System;
+    using System.IO;
+
+    public static class LogFactory
     {
-        public BaseLogger CreateLogger(string className)
+        private static string? FilePath { get; set; }
```

Comment:
```suggestion
        private static string? _FilePath;
```
Typically you don't have private auto-properties, just use a variable.